packages/components/src/components/editor/browser/browser.tsx

```diff
@@ -36,7 +37,8 @@ export const useBrowser = (
   data: Array<BrowserNode>,
   loadChildren?: (row: Row<BrowserNode>) => void,
   initialSearch?: string,
```

Comment:
Perhaps it now makes sense to create an "options" object instead of passing each option individually.

Reply:
Yes look below, you were faster 😆